app/grandchallenge/reader_studies/models.py

```diff
@@ -1164,6 +1165,7 @@ class Answer(UUIDModel):
     answer = JSONField()
     is_ground_truth = models.BooleanField(default=False)
     score = models.FloatField(null=True)
+    renditions = HistoricalRecords(related_name="history")
```

Comment:
Why not call this `history`, would be consistent with the other places that it's used.